app/pages/household/[houseId]/index.tsx

```diff
@@ -101,10 +140,68 @@ const Login = () => {
             Household {value?.data()?.name}
           </h1>
         </div>
-        <div className="basis-1/4">
-          <h1 className="text-center font-black text-2xl mb-2">Users</h1>
+        <div className="dropdown dropdown-right">
+          <label tabIndex={0} className="btn btn-circle btn-ghost btn-xs">
+            <svg
```

Comment:
The div for Users is gonna be needed for the Users list (in another PR), maybe include the div in the same div where the Household name (h1 element) is located.